src/EventTickets/resources/admin/components/EventDetailsPage/TicketTypesSection/TicketTypesRowActions.tsx

```diff
@@ -15,7 +15,7 @@ export function TicketTypesRowActions({tickets, setTickets, openEditModal}) {
                 id,
                 title,
                 description,
-                price: price / 100,
+                price: (price / 100).toFixed(2),
```

Comment:
Does this account for non-decimal currencies?

Reply:
This is not technically necessary so I just removed it on https://github.com/impress-org/givewp/pull/7346/commits/e5a383d7ecb73747e29f7e4b5048556f2539d8b6.